src/protocol/jms/src/main/java/org/apache/jmeter/protocol/jms/control/gui/JMSPropertiesPanel.java

```diff
@@ -196,7 +196,7 @@ public JPanel createPropertiesPanel() {
         JPanel panel = new JPanel(new BorderLayout(0, 5));
         panel.setBorder(BorderFactory.createTitledBorder(
                 JMeterUtils.getResString("jms_props"))); //$NON-NLS-1$
-        panel.add(GuiUtils.emptyBorder(new JScrollPane(jmsPropertiesTable)));
+        panel.add(new JScrollPane(jmsPropertiesTable));
```

Comment:
I think this should be aligned somehow.

It is true Darklaf's `titledBorder` does not render `left-right-bottom` borders, so the tables inside the panel with titled border look better with their own border.

However, other lafs produce too much borders (one for titled, another for scrollpane border) :-/

Reply:
There are ~9 usages of `GuiUtils.emptyBorder`

Reply:
I'll remove it and create a solution for darklaf, where tables has a border that paints if the surrounding scroll pane has an empty border.

Reply:
Frankly speaking, I'm not quite sure how to approach this. `GuiUtils.emptyBorder` looks like a hack, and `tables has a border that paints if the surrounding scroll pane` looks like a hack as well :-/

Reply:
The problem is that its very difficult to achieve the same visual layout with every laf as each has a different approach to borders (e.g. Nimbus has very large lowered borders that should be used sparsely).

My approach to borders when creating darklaf was that components should always provide borders on the "inwards" direction (i.e. menubar has border on bottom, tabbedPanes provide top borders for their content, splitpanes provide left/right(top/bottom) borders for their children etc...). But no non standalone (textfield, combobox,...) component provides any border to the outside itself. This prevents any double borders from occurring. The exception here is a table inside a scrollpane as it is very common and manual border setup would be tedious. 

For now I'll remove all border adjustment for tables/tabbedpanes as there needs to be another solution to deal with this.

Reply:
> The problem is that its very difficult to achieve the same visual layout with every laf as each has a different approach to borders

That is very true. I'm inclined we should probably settle on a single LaF, and code with that LaF in mind.

I think we'll release 5.3 as is (just to avoid changing too many things in a single release), then we consider dropping Nimbus, Motif (and probably others) from the LaF menu.